docs/trusted-resources.md

```diff
@@ -91,3 +91,36 @@ metadata:
 data:
   publickeys: "/etc/verification-secrets/cosign.pub, /etc/verification-secrets/cosign2.pub"
 ```
+
+#### Config key at VerificationPolicy
+VerificationPolicy supports SecretRef or encoded public key data.
+
+**Note:** Trusted Resources will first look up `VerificationPolicy` in the taskrun/pipelinerun's namespace, so the namespace should be the same of corresponding resources' namespace. If no `VerificationPolicy` it will check the system namespace, by default it is `tekton-pipelines`. If still no policy is found, it will try to fetch keys from configmap as described in last section.
+
+```yaml
+apiVersion: tekton.dev/v1alpha1
+kind: VerificationPolicy
+metadata:
+  name: verification-policy
+  namespace: tekton-pipelines
+spec:
+  resourceAuthorities:
+      # pattern is used to filter out remote resources by their sources URL. e.g. if the resources from https://github.com/tektoncd/catalog.git, then the pattern should be set to https://github.com/tektoncd/catalog.git. ".*" will match any resources.
```

Comment:
what is the format of this pattern? i think it is regex? can we link to some docs users can refer to so they know what they can put in this field?

(as you add more detail to these comments they might get a little long - it might be clearer to add this text outside of the example)

Reply:
yes, regex. will update the doc